src/main/java/com/kyu/chess/Game.java

```diff
@@ -0,0 +1,22 @@
+package com.kyu.chess;
+
+import java.util.Scanner;
+
+public class Game {
+    Scanner scanner = new Scanner(System.in);
+
+    public void start() {
+        System.out.println("start? y or n");
+        while (selectedCommandToStartGame()) {
+            Board board = new Board();
+            board.initialize();
+            System.out.println("start? y or n");
+            // TODO: 체스말 액션
+        }
```

Comment:
```suggestion
        if (!selectedCommandToStartGame()) {
            return;
        }
        Board board = new Board();
        board.initialize();
        System.out.println("start? y or n");
        // TODO: 체스말 액션
```

혹시 이게 의도하신 로직이 아닌지 여쭈어봅니다.
start 를 y 를 입력하면, 무한히 게임을 초기화할 수 있다는게 잘 이해가 가질 않아서요.
y 를 계속 누르는 한, 네버엔딩 게임이 되도록 하는게 큐가 의도하신게 맞나요?

n 을 눌렀을때, 게임이 진행되지 않고 종료되도록 하는게 Kyu 의 의도가 아닐까 해서 여쭈어봅니다.

Reply:
게임을 시작하고 난뒤에 무한히 게임을 초기화하기보다
여러가지 액션들이 취해질거같아서 일단은 y를 누르면 계속해서 체스판이 나오기 때문에,
게임을 진행한다는 느낌이어서 그렇게 했는데요. 사실..그렇게 생각하고 작성한건 아니었어요
말씀해주신거 보니 수정해주신 코드가 제가 의도하려던게 맞는거 같습니다 ㅎㅎ 감사합니다